src/components/picker/NativePicker.js

```diff
@@ -59,9 +64,11 @@ class NativePicker extends BaseComponent {
 
   renderPickerDialog = () => {
     const {selectedValue, showDialog} = this.state;
-
+    
     return (
       <PickerDialog
+        height={this.PICKER_HEIGHT + this.MENU_HEIGHT}
+        pickerStyle={{width: '100%'}}
```

Comment:
Not sure this is required.. 
usually Views are stretched horizontally (unless you change their flexDirection to be row)
So the width actually determined by the Dialog width